src/test/java/io/vertx/ext/apex/handler/StaticHandlerTest.java

```diff
@@ -133,7 +133,9 @@ public void testContentHeadersSet() throws Exception {
       String contentType = res.headers().get("content-type");
       String contentLength = res.headers().get("content-length");
       assertEquals("application/json", contentType);
-      assertEquals(18, Integer.valueOf(contentLength).intValue());
+      // Account for CRLF if checked out on Windows
```

Comment:
Hmm, is your git setup to convert unix to windows CRLF on update? Is that a good idea?